recipe/install_nvcc.sh

```diff
@@ -53,17 +63,33 @@ then
     return 1
 fi
 
-if [[ \$(grep -q "CUDA Version ${PKG_VERSION}" \${CUDA_HOME}/version.txt) -ne 0 ]]
+if [[ -z "\$(\${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release ${PKG_VERSION}")" ]]
 then
-    echo "Version of installed CUDA didn't match package"
-    return 1
+  echo "Version of installed CUDA didn't match package"
+  return 1
 fi
```

Comment:
Does this need to change?

Reply:
There was a comment earlier that `version.txt` doesn't exist properly in all CUDA installations

Reply:
Ah ok. Sorry for missing that then